<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>스레드 테스트</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>스레드 시작</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txbLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
